src/test/java/com/codingbat/exercise1/Problem3_ParrotTroubleTest.java

```diff
@@ -0,0 +1,31 @@
+package com.codingbat.exercise1;
+
+import static org.hamcrest.CoreMatchers.equalTo;
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.is;
+
+import org.junit.Before;
+import org.junit.Test;
+
+public class Problem3_ParrotTroubleTest {
+    private Problem3_ParrotTrouble fixture;
+
+    @Before
+    public void setup() {
+        fixture = new Problem3_ParrotTrouble();
+    }
+
+    @Test
+    public void Problem3_ParrotTroubleWhenParrotTalkingAndHourLessThan7OrMoreThan20() {
+        boolean expected = true;
+        boolean actual = fixture.parrotTrouble(true, 6);

```

Comment:
Can we move value 6 into a separate variable which can signify the hour value correctly and makes it more readable?